src/igvfd/schemas/treatment.json

```diff
@@ -245,7 +281,11 @@
             "type": "string",
             "enum": [
                 "Celsius"
-            ]
+            ],
```

Comment:
I think example here goes above with "temperature", need additional example here for "temperature_units"